runtime/vimrc_example.vim

```diff
@@ -1,21 +1,17 @@
 " An example for a vimrc file.
 "
 " Maintainer:   Bram Moolenaar <Bram@vim.org>
-" Last change:  2014 Nov 05
+" Last change:  2015 Jun 29
 "
 " To use it, copy it to
 "     for Unix:  ~/.vimrc
 "     for MS-DOS and Win32:  $VIM\_vimrc
```

Comment:
MS-DOS should be removed.
Since Win32 is the API, not the platform, `Windows` (or something) would be better, IMO.